etna/core/models/mixins.py

```diff
@@ -160,31 +159,14 @@ class Meta:
     ]
 
     api_fields = [
-        # APIField("hero_image"),
         APIField("hero_image_caption", serializer=RichTextSerializer()),
         APIField(
-            "hero_image_jpg",
-            serializer=ImageRenditionField(
-                "fill-1200x480|format-jpeg|jpegquality-60", source="hero_image"
-            ),
+            "hero_image",
+            serializer=DetailedImageSerializer("fill-1200x480"),
         ),
         APIField(
-            "hero_image_webp",
-            serializer=ImageRenditionField(
-                "fill-1200x480|format-webp", source="hero_image"
-            ),
-        ),
-        APIField(
-            "hero_image_jpg_small",
-            serializer=ImageRenditionField(
-                "fill-600x400|format-jpeg|jpegquality-60", source="hero_image"
-            ),
-        ),
-        APIField(
-            "hero_image_webp_small",
-            serializer=ImageRenditionField(
-                "fill-600x400|format-webp", source="hero_image"
-            ),
+            "hero_image_small",
+            serializer=DetailedImageSerializer("fill-600x400", source="hero_image"),

```

Comment:
```suggestion
            serializer=ImageSerializer("fill-600x400", source="hero_image"),

```

Again, not all the fields are needed.